src/sql/workbench/contrib/notebook/browser/notebookViews/cellToolbar.css

```diff
@@ -0,0 +1,53 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.actionbar {
```

Comment:
I don't see this css file being imported anywhere - are you sure it's being used?

If it is then you should also be making sure that the styles are scoped down to the component level correctly.

Reply:
I had broken up a previous PR and omitted the import by mistake. I have added the import and scoped the styles down to the component.